### PR TITLE
Add timestamps to program as useful output

### LINKED ISSUE #13: Create the testing structure to compare algorithm implementations
Brainstorm ways to compare the two (or more) algorithm implementations. This could be comparing:
1. Runtime, 
2. Complexity of the algorithm to understand,
3. Complexity of the algorithm to implement,
4. Difficulty in making the training data for the model,
5. How well the model solves our problem (does it predict just one disease? How about multiple?)
6. Accuracy of the algorithm against the test data noted in #8 (for example)
etc.

Make a table with the necessary columns and rows to compare algorithms and add it to the final report, maybe even fill out the table if the algorithms are implemented and ready to compare. Think about the conclusions we could derive from such data.

### PR DESCRIPTION
**Pull request by holtzmak**
Closes #13 . Fancy output includes:
```
% python3 medi_app.py -bayesnet_from_file Example_Training_Samples
Please enter your symptoms separated by spaces (e.g. "itching chills vomiting"): 
itching skin_rash           
Model finished construction in 0.033814581999997984 seconds
The predicted disease is Fungal infection with probability of 0.002776783521541043
The Bayes's Net implementation completed in 0.07208790700000378 seconds
```
and 
```
% python3 medi_app.py -ann                                       
Model finished construction in 3.468400333 seconds
The ANN reported:
[[ 0  0  0 22  0  0  0]
 [ 0  0  0 25  0  3  0]
 [ 0  0 21  0  0  0  0]
 [ 0  0  0 22  0  0  0]
 [ 0  0  0  0 29  0  0]
 [ 0  0  0  0  0 23  0]
 [ 0  0  0 21  0  0  0]]
              precision    recall  f1-score   support

           0       0.00      0.00      0.00        22
           1       0.00      0.00      0.00        28
           2       1.00      1.00      1.00        21
           3       0.24      1.00      0.39        22
           4       1.00      1.00      1.00        29
           5       0.88      1.00      0.94        23
           6       0.00      0.00      0.00        21

    accuracy                           0.57       166
   macro avg       0.45      0.57      0.48       166
weighted avg       0.46      0.57      0.48       166

The ANN Implementation completed in 3.7979925089999997 seconds
```

### CODE REVIEW

**Comment by holtzmak** [comment]
@alwinbaby sorry, not sorry... Bayes's Net is faster!

File: src/medi_app.py [hunk 1]
```diff
@@ -38,10 +39,15 @@ def get_symptoms_from_user(known_symptoms: List):
...
         elif option == "-ann":
             if __name__ == "__main__":
+                program_start = default_timer()
                 ann = ArtificialNeuralNetwork()
                 x, y = ann.data_sweep()
                 x_train, x_test, y_train, y_test = ann.ann_train_test_split(x, y)
                 ann.ann_training(x_train, x_test, y_train, y_test)
+                program_end = default_timer()
+                print(
+                    f"The ANN Implementation completed in {program_end - program_start} seconds"
```
Comment (holtzmak) [inline src/medi_app.py:49]:
```suggestion
                    f"The ANN implementation completed in {program_end - program_start} seconds"
```
